[All Hosts] (debugging) fixes in chromium debugging article (#2875)
OfficeDev/office-js-docs-pr: docs/testing/debug-desktop-using-edge-chromium.md

- [Visual Studio Code](https://code.visualstudio.com/) (must be run as an administrator)
- [Node.js (version 10+)](https://nodejs.org/)
- Windows 10

> - A combination of platfrom and Office host that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of Microsoft 365 is earlier than 2101, you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).

Reviewer: ```suggestion
- A combination of platform and Office application that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of Microsoft 365 is earlier than 2101, you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
```

## Install and use the debugger

1. Create a project using the [Yeoman generator for Office Add-ins](https://github.com/OfficeDev/generator-office). You can use any one of our quick start guides, such as the [Outlook add-in quickstart](../quickstarts/outlook-quickstart.md), in order to do this.

> [!TIP]
> If you aren't using a Yeoman generator based add-in, you may be prompted to adjust a registry key. While in the root folder of your project, run the following in the command line.

>  `office-add-in-debugging start <your manifest path>`

Reviewer: The formatting is off for this line on the review site. I think adding the > will help. 
```suggestion
 > `office-add-in-debugging start <your manifest path>`
```

> 1. Open your project in VS Code. Within VS Code, select **CTRL + SHIFT + X** to open the Extensions bar. Search for the "Debugger for Microsoft Edge" extension and install it.

Reviewer: ```suggestion
1. Open your project in VS Code. Within VS Code, select **Ctrl+Shift+X** to open the Extensions bar. Search for the "Debugger for Microsoft Edge" extension and install it.
```

> 1. Next, choose  **View > Run** or enter **CTRL + SHIFT + D** to switch to debug view.

Reviewer: Keyboard shortcuts have a first-letter capitalization rule and no space before/after the plus sign. [Style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=27401). 
```suggestion
1. Next, choose  **View > Run** or enter **Ctrl+Shift+D** to switch to debug view.
```

1. You're now able to set breakpoints in your project's code and debug.

   > [!NOTE]
   > Breakpoints in calls of [Office.initialize]() or [Office.onReady]() are ignored. For details about these methods, see [Initialize your Office Add-in](../develop/initialize-add-in.md).

> [!IMPORTANT]

> > The best way to stop a debugging session is to select **Shift-F5** or choose **Run > Stop Debugging** from the menu. This action should close the Node server window and attempt to close the host application, but there will be a prompt on the host application asking you whether to save the document or not. Make an appropriate choice and let the host close. Avoid manually closing the Node window or host application. Doing so can cause bugs especially when you are stopping and starting debugging sessions repeatedly.

Reviewer: ```suggestion
> The best way to stop a debugging session is to select **Shift+F5** or choose **Run > Stop Debugging** from the menu. This action should close the Node server window and attempt to close the host application, but there will be a prompt on the host application asking you whether to save the document or not. Make an appropriate choice and let the host application close. Avoid manually closing the Node window or host application. Doing so can cause bugs especially when you are stopping and starting debugging sessions repeatedly.
```
